dev-requirements.txt

```diff
@@ -7,42 +7,55 @@
 appdirs==1.4.4            # via virtualenv
 aspy.yaml==1.3.0          # via pre-commit
 atomicwrites==1.4.0       # via pytest
-attrs==19.3.0             # via pytest
+attrs==19.3.0             # via -c requirements.txt, jsonschema, pytest
 certifi==2020.6.20        # via -c requirements.txt, requests
 cfgv==2.0.1               # via -r dev-requirements.in, pre-commit
 chardet==3.0.4            # via -c requirements.txt, requests
 click==7.1.2              # via -c requirements.txt, flask
+configparser==4.0.2       # via -c requirements.txt, detect-secrets, importlib-metadata
+contextlib2==0.6.0.post1  # via -c requirements.txt, importlib-metadata, importlib-resources, zipp
 coverage==5.2             # via pytest-cov
 detect-secrets==0.13.0    # via -r dev-requirements.in
 distlib==0.3.1            # via virtualenv
+enum34==1.1.10            # via detect-secrets
```

Comment:
This might need special handling too as it is a backport for py3.3 down to py27

Reply:
good catch.  it's not breaking the build.  but i'll make it conditional anyway.